src/azul/service/manifest_service.py

```diff
@@ -1005,8 +1005,10 @@ def source_filter(self) -> SourceFilters:
     @classmethod
     def command_lines(cls, url: str) -> JSON:
         return {
-            'cmd.exe': f'curl.exe --location {cls._cmd_exe_quote(url)} | curl.exe --config -',
-            'bash': f'curl --location {shlex.quote(url)} | curl --config -'
+            # The -f flag allows curl to fail silently on server errors, and
+            # instead exit with error 22, followed by the HTTP status code.
```

Comment:
```suggestion
            # The -f flag will allow curl to fail silently on server errors, returning
            # the HTTP status code to STDERR instead of returning the full error
            # response from the server to STDOUT.
```